zenoh/src/net/routing/dispatcher/pubsub.rs

```diff
@@ -133,7 +133,7 @@ pub(crate) fn undeclare_subscription(
             Some(prefix) => match Resource::get_resource(prefix, expr.suffix.as_ref()) {
                 Some(res) => Some(res),
                 None => {
-                    tracing::error!(
+                    tracing::trace!(
```

Comment:
This one should not occur with proper ACL (expo is not empty). Please revert to error.